src/components/CodeRegistrationModal/CodeRegistrationModal.jsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react';
+import OtpInput from 'react-otp-input';
+import styles from './CodeRegistrationModal.module.scss'
+import iconQuestion from "../../images/iconQuestion.png"
+import arrow from '../../images/arrow.svg';
+
+export default function CodeRegistrationModal({ onVerify, onSkip, handleBack }) {
+  const [code, setCode] = useState("");
+
+  const handleChange = (code) => {
+    setCode(code);
+  }
+  const pattern = /^\d{6}$/;
+
+  const handleVerify = () => {
+    if (pattern.test(code)) {
+      onVerify(code)
+    } else {
+      setCode("");
+      console.log('код повинен складатись з 6 цифр')
+    }
+  }
```

Comment:
imagine, user entered 5 digits out of of 6 and making mistake in the last symbol and typing a alphabetical character in, like O instead of 0.
then click "submit" button and gets the whole form losing its input.
and then similar situation repeats, coz user has Parkinson disease and their hands are utterly shaking.
what a frustrating experience it must be to fill the code in again and again for them.  

so let's be a bit more user friendly and what if instead of resetting the code we just do not allow to enter any alphabetical symbols?

here is an example: https://ej2.syncfusion.com/react/documentation/otp-input/accessibility

Reply:
You are absolutely right, I fixed it